src/web/components/ApiKeyManagement/KeyItem.tsx

```diff
@@ -0,0 +1,25 @@
+import { ApiKeyDTO } from '../../../api/services/adminServiceHelpers';
+import ApiRolesCell from './ApiRolesCell';
+
+function formatDate(timeValue: number) {
```

Comment:
Two things: 
1. Could this method possibly be exported from a more generic utils folder, or is this specific to KeyItems?
2. Could we possibly avoid magic numbers and extract `1000` to a variable which explains what it is?

Reply:
Moved the value to a constant. I'm not sure it's worth moving to a generic. It's somewhat specific since it's receiving a Unix timestamp in seconds which is always a given.

Reply:
Unix timestamps are used all over admin - perhaps it is indeed worth extracting out and renaming to `formatUnixDate`

Reply:
Sounds good. Have moved to the function to ```textHelpers.ts```.